packages/side-nav/src/SideNavItem.tsx

```diff
@@ -1,132 +1,108 @@
 import React, { ReactNode } from 'react';
 import PropTypes from 'prop-types';
-import omit from 'lodash/omit';
-import { css, cx } from '@leafygreen-ui/emotion';
-import { uiColors } from '@leafygreen-ui/palette';
-import { AriaCurrentValue, createDataProp } from '@leafygreen-ui/lib';
+import { transparentize } from 'polished';
+import {
+  AriaCurrentValue,
```

Comment:
should we move this to a11y

Reply:
Maybe, but in a different PR. Where this lives doesn't meaningfully impact the implementation of this PR, so it's out of scope.

Reply:
Makes sense to me, let's just make a ticket to move it then?